kafka-gateway/pom.xml

```diff
@@ -46,14 +46,28 @@
         </dependency>
         <dependency>
             <groupId>za.co.absa.spline</groupId>
-            <artifactId>producer-model-v1.1</artifactId>
+            <artifactId>producer-model-mapper</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <!-- current model -->
+
         <dependency>
             <groupId>za.co.absa.spline</groupId>
-            <artifactId>producer-model-mapper</artifactId>
+            <artifactId>producer-model-v1.2</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <!-- deprecated models -->
+
+        <dependency>
+            <groupId>za.co.absa.spline</groupId>
```

Comment:
Why still keeping this?

Reply:
Both Kafka and REST endpoints of Spline Producer API provide backward compatibility with some of the older models via the model mapper. The concrete models are defined as optional dependencies for the mapper, thus the client modules have to explicitly declare what models they actually support.